C#/multicorp-bot/Models/DbModels/Loans.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace multicorp_bot
+{
+    public partial class Loans
+    {
+        public int LoanId { get; set; }
+        public int ApplicantId { get; set; }
+        public string Status { get; set; }
+        public int? FunderId { get; set; }
+        public long? OrgId { get; set; }
+        public long? RequestedAmount { get; set; }
+        public long? InterestAmount { get; set; }
+        public long? RemainingAmount { get; set; }
+        public int IsCompleted { get; set; }
```

Comment:
Why int and not bool?

Reply:
Holdover from MsSQL since it doesn't have a book type. Generally it's a bit value though